app/controllers/api/v1/blocks_controller.rb

```diff
@@ -18,6 +18,8 @@ def load_accounts
 
   def paginated_blocks
     @paginated_blocks ||= Block.eager_load(target_account: :account_stat)
+                               .joins(:target_account)
+                               .merge(Account.without_suspended)
```

Comment:
This is *probably* fine in the blocks controller, but as above, I would prefer if an un-suspended user could still manage their relationships with a suspended user.